_layouts/wiki.html

```diff
@@ -15,7 +15,6 @@
     <div id="maincontainer" class="fx-col-100-xs fx-col-50-m">
       <noscript class="red"><strong>{{ site.data.general.tNoJSEnabled }}</strong></noscript>
       <main lang="{{ page.lang }}">{{ content }}</main>
-      <div id="ftext" class="fx-txt-center">
```

Comment:
Only one div removed? Shouldn't the closing div also be removed (including the CSS id)?